docs/how-to/build-a-12-factor-app-rock.rst

```diff
@@ -0,0 +1,165 @@
+How to build a 12-Factor app rock
+*********************************
+
+The following how-to guides provide instructions on managing
+and configuring rocks for 12-factor applications.
+
+Include extra files in the OCI image
+------------------------------------
+
+The following files are included in the image by default from
+the root of the project:
+
+- ``app`` (does not apply to the ``go-framework``)
+- ``app.py`` (does not apply to the ``go-framework``)
+- ``migrate``
+- ``migrate.sh``
+- ``migrate.py`` (does not apply to the ``go-framework``)
+- ``static``
+- ``templates``
+
+To change this, the following snippet needs to be added to
+the ``rockcraft.yaml``:
+
+.. tabs::
+
+   .. group-tab:: Flask
```

Comment:
This looks broken on the document:
![image](https://github.com/user-attachments/assets/ef89b967-ceac-4fbb-a564-a629e5f43283)

I think it's because these first three group-tabs have 3 spaces at the beginning, but the `.. group-tab:: Go` on has 2

Reply:
Thanks for catching! The tabs are rendering correctly on my end now. Let me know whether you're still seeing issues on your end